Shared/Extensions/JellyfinAPI/DayOfWeek.swift

```diff
@@ -0,0 +1,32 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import Foundation
+import JellyfinAPI
+
+extension DayOfWeek {
+
+    var displayTitle: String {
+        switch self {
+        case .sunday:
+            return L10n.dayOfWeekSunday
+        case .monday:
+            return L10n.dayOfWeekMonday
+        case .tuesday:
+            return L10n.dayOfWeekTuesday
+        case .wednesday:
+            return L10n.dayOfWeekWednesday
+        case .thursday:
+            return L10n.dayOfWeekThursday
+        case .friday:
+            return L10n.dayOfWeekFriday
+        case .saturday:
+            return L10n.dayOfWeekSaturday
```

Comment:
These values are already localized by Apple as a part of Calendar. Something like the below should work:
```
func localized() -> String? {
    let newLineRemoved = self.rawValue.replacingOccurrences(of: "\n", with: "")
    
    guard let index = DateFormatter().weekdaySymbols.firstIndex(of: newLineRemoved) else {
        return nil
    }

    let localCal = Calendar.current
    return localCal.weekdaySymbols[index].localizedCapitalized
}
```

Reply:
I do not know why there is a new line character coming from the enum btw. But I discovered it when testing in Playgrounds.